cmd/golada/main.go

```diff
@@ -1,22 +1,22 @@
-// Copyright © 2019 The Homeport Team
+//Copyright © 2019 The Homeport Team
```

Comment:
Any chance to get GoLand to give me that beloved space after the double slash?

Reply:
GoLand can do whatever your heart desiers ;)